### PR TITLE
Fixed File target as . in nupkg with Pack command 

### PR DESCRIPTION
**Pull request by jainaashish**
File with target as . goes into . folder in nupkg with Pack command which should go into root folder instead

Fix https://github.com/NuGet/Home/issues/2937

@emgarten @joelverhagen @drewgil @rrelyea

### CODE REVIEW

File: test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageBuilderTest.cs [hunk 1]
```diff
@@ -2518,6 +2518,33 @@ var contentTypesReader
             }
         }
 
+        [Fact]
+        public void PackageBuilderWorksWithFilesHavingCurrentDirectoryAsTarget()
```
Comment (joelverhagen) [inline test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageBuilderTest.cs]: Can we test other similar cases? e.g. `./blah/../foo.txt`, `.\blah\..\foo.txt`, `..\foo.txt`, `foo\.\.\.\bar\..\bar.txt`.

**Comment by joelverhagen** [comment]
Looks good, but adding more edge cases to the tests would be good. :watch:

File: test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageBuilderTest.cs (hunk 1 above)
Comment (jainaashish) [inline test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageBuilderTest.cs]: the one added was a valid case to define target as "." but defining "./blah/../foo.txt" or similar stuff should not be advised and we can always ask them to correct these targets. And that's why i didn't add tests around that

Comment (joelverhagen) [inline test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageBuilderTest.cs]: Whether or not this is a recommended case, we should test and assert what happens. Users can come to depend on these sort of formats and, in the future, we should know when the behavior is changing (and potentially choose _not_ to change the behavior).

**Comment by joelverhagen** [comment]
:shipit:

**Comment by jainaashish** [comment]
Merged 2715815f1207c146897cd9a925d27331347da73c

**Pull request closed without merging**